phpstan-baseline.neon

```diff
@@ -0,0 +1,407 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Method Keboola\\\\CsvMap\\\\Exception\\\\CsvMapperException\\:\\:getData\\(\\) return type has no value type specified in iterable type array\\.$#"
```

Comment:
nezkoumal jsem to, ale proč nemají nastavené typy?

Reply:
Nevím, nebylo to tam a nemyslím si, že je ve scopu abych to dával v rámci přepisu Monga do kupy. Proto jsem vygeneroval baseline ať jsou aspoň nový commity zkontrolovaný a tyhle chyby, co tam byly se buď vyřeší až se to do toho bude sahat nebo na to vytvoříme task, ne?

Reply:
https://keboola.atlassian.net/browse/COM-1483